src/ensembl/src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -458,6 +508,45 @@ const statsFormattingOptions: StatsFormattingOptions = {
       label: 'Average GC content'
     }
   },
+  [SpeciesStatsSection.HOMOLOGY]: {
+    [Stats.HOMOLOGY]: {
+      label: 'Homology'
+    },
+    [Stats.HOMOLOGY_COVERAGE]: {
+      label: 'Coverage',
+      headerUnit: 'coverage',
+      primaryValuePostfix: '%'
+    }
+  },
+  [SpeciesStatsSection.VARIATION]: {
+    [Stats.VARIATION]: {
+      label: 'Variation'
+    },
+    [Stats.SHORT_VARIANTS]: {
+      label: 'Short variants',
+      headerUnit: 'short variants'
+    },
+    [Stats.STRUCTURAL_VARIANTS]: {
+      label: 'Structural variants',
+      headerUnit: 'structural variants'
+    },
+    [Stats.SHORT_VARIANTS_WITH_PHENOTYPE_ASSERTIONS]: {
+      preLabel: 'Short Variants',
+      label: 'With phenotype assertions'
+    },
+    [Stats.SHORT_VARIANTS_WITH_PUBLICATIONS]: {
+      preLabel: 'Short Variants',
+      label: 'With publications'
+    },
+    [Stats.SHORT_VARIANTS_FREQUENCY_STUDIES]: {
+      preLabel: 'Short Variants',
+      label: 'Frequency studies'
+    },
+    [Stats.STRUCTURAL_VARIANTS_WITH_PHENOTYPE_ASSERTIONS]: {
+      preLabel: 'Structural Variants',
```

Comment:
Why does the same text, for the same label, sometimes appear with both words starting with a capital letter (Short Variants, Structural Variants), and other times with only the first word starting with a capital (Short variants, Structural variants)?